microcosm_secretsmanager/loaders/loaders.py

```diff
@@ -0,0 +1,18 @@
+from os import environ
+
+from microcosm.loaders import load_from_dict
+
+
+class VersionedSecretsManagerLoader:
```

Comment:
???
This is like a copy-paste of the previous file, without any of the useful bits.